src/utils.ts

```diff
@@ -54,26 +54,31 @@ export function getSingleProperty(
  */
 export function checkCollectionName(collectionName: string): void {
   if ('string' !== typeof collectionName) {
+    // TODO(NODE-3405): Use MongoNamespace static method
```

Comment:
We should probably make a ticket for this so we don't forget what exactly this refers to